.github/ISSUE_TEMPLATE/new-ontology-request.md

```diff
@@ -50,6 +50,19 @@ GitHub username:
 
 ## Additional comments or remarks
 
+## OBO Foundry pre-release checklist
+
```

Comment:
Add some introductory text here, e.g., "To be considered for inclusion in the OBO Foundry, an ontology must meet certain requirements, as described at https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md. Please review the checklist below and ensure that you have met all these requirements."